engines/bastion_katello/lib/bastion_katello/engine.rb

```diff
@@ -33,6 +33,7 @@ class Engine < ::Rails::Engine
         ),
         :config => {
           'consumerCertRPM' => consumer_cert_rpm,
+          'enable_download_policies' => !Foreman.in_rake? && Setting[:enable_download_policies],
```

Comment:
actually I was missing it from [here](https://github.com/Katello/katello/pull/5939/files#diff-9985f2990a24b787bc476884bf049ab1R5) :smile: